apps/voting/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconLock,
+  Info,
+  textStyle,
+  Timer,
+  useTheme,
+} from '@aragon/ui'
+import { getAgreement } from '../../agreementsMockData'
+import { addressesEqual } from '../../web3-utils'
+
+function DisputableActionStatus({ vote, connectedAccount }) {
+  //TODO: get agreement and vote real data
+  const theme = useTheme()
+  const agreement = getAgreement()
+
+  return (
+    <React.Fragment>
+      <Box heading="Disputable Action Status">
+        <Item>
+          <Label>Status</Label>
+          <div>DisputableStatusTag</div>
+        </Item>
+        <Item>
+          <Label>Action collateral locked</Label>
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+            `}
+          >
+            {vote.disputable.action.collateral.challengeAmount}{' '}
+            {vote.disputable.action.collateral.collateralToken}
+            <span
+              css={`
+                padding-left: ${1 * GU}px;
```

Comment:
```suggestion
                display: inline-flex;
                padding-left: ${1 * GU}px;
```

Should make this icon center nicely against the text